README.md

```diff
@@ -57,17 +57,4 @@ A: [Here](https://github.com/getblocklab/block-lab/wiki)
 
 ## Contributing ##
 
-First execute...
-```
-npm install
-```
-
-For development...
-```
-npm run dev 
-```
-
-For build...
-```
-npm run build 
-```
+See [CONTRIBUTING.md](CONTRIBUTING.md)
```

Comment:
Can we please use the full URL here as this file gets converted and packaged for the WordPress.org directory.